en/flight_modes/acro_fw.md

```diff
@@ -0,0 +1,39 @@
+# Acro Flight Mode (Fixed Wing)
+
+<!-- this requires review and updates -->
+
+[<img src="../../assets/site/difficulty_5.svg" title="Difficulty (Hard)" width="30px" />](../getting_started/flight_modes.md#key_difficulty)&nbsp;[<img src="../../assets/site/remote_control.svg" title="Manual/Remote control required" width="30px" />](../getting_started/flight_modes.md#key_manual)&nbsp;
+
+*Acro mode* is the RC mode for performing acrobatic maneuvers e.g. rolls, flips, stalls and acrobatic figures.
+
+The roll, pitch and yaw sticks control the rate of angular rotation around the respective axes and throttle is passed directly to the output mixer. When sticks are centered the vehicle will stop rotating, but remain in its current orientation (on its side, inverted, or whatever) and moving according to its current momentum.
+
+![FW Manual Acrobatic Flight](../../images/flight_modes/manual_acrobatic_FW.png)
+
+## Technical Description
+
+RC mode for performing acrobatic maneuvers e.g. rolls, flips, stalls and acrobatic figures.
+
+RPY stick inputs are translated to angular rate commands that are stabilized by autopilot. 
+Throttle is passed directly to the output mixer.
+
+
+## Parameters
+
+Parameter | Description
+--- | ---
+&nbsp; | 
+
+
+<!--

```

Comment:
Below are possibly relevant parameters. Should these be made visible? Are there any others?

Reply:
This is a good set. In acro it's particularly important to know your maximum rates and stick curve (only linear for fw at the moment, we might add expo in future, as we have for mc)

Reply:
Thanks. Will add!